src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -127,6 +131,8 @@ public class PatchDetailsPanel : PanelContainer
     private Patch? targetPatch;
     private Patch? currentPatch;
 
+    private bool moveToPatchButtonVisible;
```

Comment:
The default value should be true.

Reply:
This will now have a default value of true.